sirepo/simulation_db.py

```diff
@@ -263,9 +263,7 @@ def get_schema(sim_type):
     pkcollections.mapping_merge(
```

Comment:
Could this just be schema.feature_config = ...

Reply:
https://github.com/radiasoft/sirepo/pull/2532/commits/e8d7a943f378eb6fdb6a9315cea4522022b14197